packages/iproute/iproute.spec

```diff
@@ -46,7 +45,7 @@ export PKG_CONFIG_PATH='%{_cross_pkgconfigdir}' \\\
 %make_install
 
 install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
-mv %{buildroot}%{_cross_sysconfdir}/iproute2 %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
+mv %{buildroot}%{_datarootdir}/iproute2 %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
```

Comment:
If `ip` will look for its files in `/usr/share` instead of `/etc`, that would be great - saves time copying stuff to tmpfiles. There is probably an `export SOMETHINGDIR='%{_cross_datadir}'` that would get `make install` to put stuff in the right place.

(Does it still look in `/etc`? If not then this change is not correct.)

Reply:
Yeah. [This](https://github.com/iproute2/iproute2/commit/962692356a1cc41932e43575f3c50927e455ab53) is the change I was using to make this change. They're explicit about using `/usr/share` instead of `/etc` from my reading

Reply:
Running into errors when trying the suggested changes. I'm removing this update from this PR to take a closer look